lib/movie_main_widget.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'text_styles.dart';
+import 'genres.dart';
+import 'movie.dart';
+import 'movie_title.dart';
+import 'top_row.dart';
+
+class MovieMainWidget extends StatelessWidget {
+  const MovieMainWidget({
+    super.key,
+    required this.movie,
+  });
+
+  final Movie movie;
+
+  static const EdgeInsets topRowPadding = EdgeInsets.only(
+    top: 150,
+    left: 10,
+  );
+
+  static const EdgeInsets movieGenresPadding = EdgeInsets.only(top: 30);
+  static const EdgeInsets movieTitlePadding = EdgeInsets.only(top: 35);
+  static const EdgeInsets movieOverviewPadding = EdgeInsets.only(top: 35);
```

Comment:
Separate this as a mentioned before, also you have movieTitlePadding and movieOverviewPadding that has exactly the same values

Reply:
Addressed